Gruntfile.js

```diff
@@ -70,6 +70,12 @@ module.exports = function(grunt) {
                 return "url('" + imgUrl + "') repeat";
               }
               return 'initial';
+            },
+            assignFontSize: function(fontSize) {
+              if (fontSize) {
+                return fontSize + 'rem' || 'initial';
```

Comment:
Ok I made the changes in my fork. So how does this work, do I make another pull request or does this one stay open?